src/app/cmdoptions.cpp

```diff
@@ -481,6 +484,9 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
         }
     }
 
+    if (QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR))
```

Comment:
IMO, it's not "Command line options" job.

Reply:
Agreed, checking some folder for existence is not related to command line parsing. I suggest to do that after command line parsing

Reply:
Do you mean it's better to add folder check here: https://github.com/qbittorrent/qBittorrent/blob/565eb4cdc8631ce0e8a999bd1234d5ce67baf990/src/app/application.cpp#L149 ?

Should I also add folder checks here:
https://github.com/qbittorrent/qBittorrent/blob/565eb4cdc8631ce0e8a999bd1234d5ce67baf990/src/app/application.cpp#L743 and here:
https://github.com/qbittorrent/qBittorrent/blob/565eb4cdc8631ce0e8a999bd1234d5ce67baf990/src/app/application.cpp#L746 or these two places are ok to check only for `m_commandLineArgs.portableMode`?

Reply:
just do it in only one place, right exactly before the line you posted first. just set `m_commandLineArgs.portableMode` to `true` in case if folder exists, and leave everything as is. it must work

Reply:
>just set m_commandLineArgs.portableMode to true

Spoofing command line parameters doesn't seem like the better solution. Ideally, it should be checked in the place where the final decision is made.

Reply:
I think it's better to set `m_commandLineArgs.portableMode` to `true` before https://github.com/Tester798/qBittorrent/blob/e9e30993eebbc7f678af3632a604a7584b2df598/src/app/application.cpp#L150 because in this case we can still get useful messages from https://github.com/Tester798/qBittorrent/blob/e9e30993eebbc7f678af3632a604a7584b2df598/src/app/application.cpp#L746-L750
Updated pull request.